modules/route_replies.cpp

```diff
@@ -369,7 +369,10 @@ class CRouteRepliesMod : public CModule
 		// The timer will be deleted after this by the event loop
 
 		if (!GetNV("silent_timeouts").ToBool()) {
-			PutModule("This module hit a timeout which is possibly a bug.");
+			PutModule("This module hit a timeout which is possibly a " +
```

Comment:
C++ does not allow concatenating raw string literals ("foo" + "bar"). Either write the whole message on the same line, or leave out the '+' mark. I'd prefer the former, because if you choose the latter, then the indentation of the second line also needs to be fixed.

    PutModule("This module hit a timeout which is possibly a connectivity issue.");